code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -90,6 +90,7 @@
 			qdel(S)
 			return cost * (spell_levels+1)
 	return -1
+

```

Comment:
run `git checkout upstream/master -- code/modules/antagonists/wizard/equipment/spellbook.dm` to totally undo all changes to this file